### PR TITLE
acm4471 add disco clust upgr policy 2.8

### PR DESCRIPTION
**Pull request by cadawson**
please put the issue # on the commit message, like we did with GH. You can simply add the URL: 

https://issues.redhat.com/browse/ACM-4471

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @elgnay. This is the policy version of the documentation to upgrade a cluster. See if this looks ok, and we can backport to 2.7. Thank you!

**Comment by cadawson** [comment]
Hi @elgnay. Can you take a look at this PR when you get a chance? It is the policy version of the disconnected cluster upgrade. Thank you!

File: install/upgrade_cluster_disconnected_policies.adoc [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+. From the hub cluster CLI, enter the `oc get pods` command to view the status of the instance creation. It might take a while, but the process is complete when the result of the command shows that the instance and the operator are running.
+
+[#deploy-a-policy-to-override-the-default-registry]
+== Deploy a policy to override the default registry (optional)
+
+*Note:* The steps in this section only apply if you have mirrored your releases into your mirrored registry. 
+
+{ocp-short} has a default image registry value that specifies where it finds the upgrade packages. In a disconnected environment, you can create a policy to replace that value with the path to your local image registry where you mirrored your release images. 
+
+For these steps, the policy is named _ImageContentSourcePolicy_. Complete the following steps to create the policy:
```
Comment (elgnay) [inline install/upgrade_cluster_disconnected_policies.adoc]: rename `_ImageContentSourcePolicy_ `to `_policy-mirror_`

File: install/upgrade_cluster_disconnected_policies.adoc [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+
+. In the {ocp-short} navigation, select *Administration* > *Custom Resource Definitions*.
+
+. Select the _Instances_ tab.
+
+. Select the name of the _ImageContentSourcePolicy_ that you created when you set up your disconnected OLM to view the contents.
+
+. Select the _YAML_ tab to view the content in YAML format.
+
+. Copy the entire contents of the ImageContentSourcePolicy.
```
Comment (elgnay) [inline install/upgrade_cluster_disconnected_policies.adoc]: We don't need those steps. Sorry for providing the redundant content.

File: install/upgrade_cluster_disconnected_policies.adoc [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      []  # selects all clusters if not specified
+----
++
+<1> Replace the content inside the `objectDefinition` section of the template with content and add the settings for your `ImageContentSourcePolicy`.
```
Comment (elgnay) [inline install/upgrade_cluster_disconnected_policies.adoc]: I think we don't need this line as well.

File: install/upgrade_cluster_disconnected_policies.adoc [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+[#deploy-a-policy-to-deploy-a-disconnected-catalog-source]
+== Deploy a policy to deploy a disconnected catalog source
+
+Push the _Catalogsource_ policy to the managed cluster to change the default location from a connected location to your disconnected local registry. 
+
+. In the console, select *Infrastructure* > *Clusters*.
+
+. Find the managed cluster to receive the policy in the list of clusters.
+
+. Note the value of the `name` label for the managed cluster. The label format is `name=managed-cluster-name`. This value is used when pushing the policy.
```
Comment (elgnay) [inline install/upgrade_cluster_disconnected_policies.adoc]: We can remove those lines as well.

**Comment by elgnay** [comment]
The instructions described in this doc are not tested because the dev team has no disconnected env. It's necessary to ask some one to test this guide in a real disconnected env.

**Comment by cadawson** [comment]
@elgnay I updated the content with your comments. I reached out to @hchenxa and @laisongls form the QE team on 4/28 to see if they would have time to test this when they return.

**Review by laisongls — APPROVED** [review]

**Review by oafischer — APPROVED** [review]
Looks great!

**Pull request merged**